packages/GA4Client/src/common.js

```diff
@@ -1,3 +1,14 @@
+var EVENT_MAX_LENGTH = 40;
+var USER_ATTRIBUTE_MAX_LENGTH = 24;
+var EVENT_VAL_MAX_LENGTH = 100;
+var USER_ATTRIBUTE_VALUE_MAX_LENGTH = 36;
```

Comment:
I know these constants were pulled from Android's kit, but I'd go one step further to make each even more descriptive. I'd also add another constant to differentiate between `EVENT_NAME` and `EVENT_ATTRIBUTE_KEY`:

```suggestion
var EVENT_NAME_MAX_LENGTH = 40;
var EVENT_ATTRIBUTE_KEY_MAX_LENGTH = 40;
var EVENT_ATTRIBUTE_VAL_MAX_LENGTH = 100;
var USER_ATTRIBUTE_KEY_MAX_LENGTH = 24;
var USER_ATTRIBUTE_VALUE_MAX_LENGTH = 36;
```

Reply:
Then modify the below variable names of course.